### ISSUE TITLE
AttributeError: 'FigureMention' object has no attribute 'url'

### ISSUE THREAD

**Issue by HiromuHota** [HazyResearch/fonduer-tutorials#69]
**Describe the bug**

At the end cell of the hardware_image tutorial, I got the following error.

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-22-c9b701e051fd> in <module>
      4 
      5 print(train_cands[n])
----> 6 Image(f"{docs_path}/{train_cands[n][0].context.url}", format='png')

AttributeError: 'FigureMention' object has no attribute 'url'
```

**To Reproduce**
1. Run all down to the end of the hardware_image tutorial.

**Expected behavior**

Run w/o an error

**Error Logs/Screenshots**

![image](https://user-images.githubusercontent.com/20668349/79911808-db601000-83d5-11ea-80c3-4fbcfdb7ec99.png)

**Environment (please complete the following information):**
 - OS: macOS [10.15.4]
 - PostgreSQL Version: 12.1-1.pgdg100+1
 - Poppler Utils Version: 0.83.0
 - Fonduer Version: 0.8.2-dev (https://github.com/HazyResearch/fonduer/commit/a1d853f)

**Additional context**

**Comment by senwu** [comment]
It should be `Image(f"{docs_path}/{train_cands[n][0].context.figure.url}", format='png')` now. Due to this PR (https://github.com/HazyResearch/fonduer/pull/381).

**Comment by HiromuHota** [comment]
That was it! Thanks, @SenWu. I'll submit a PR for this.

**Comment by HiromuHota** [comment]
I just realized that HazyResearch/fonduer#381 was one of the changes at v0.8.0.
I'll just include a fix to #68 .
